instrumentation/opentelemetry-instrumentation-grpc/src/opentelemetry/instrumentation/grpc/_aio_client.py

```diff
@@ -43,6 +46,15 @@ def callback(call):
                         description=details,
                     )
                 )
+            if response_hook:
+                try:
+                    response_hook(span, details)
```

Comment:
Why is the repsonse hook called here in a different way than all the other hooks?

Reply:
for some reason this method is not a part of the class... so we don't have `self` here
do you think i should put it inside? 
there was probably some reason for this

Reply:
So why not make extract it from the class to the file and pass it the hook functions instead of it accessing self?

Reply:
wallak worked 👍